docs/page/css/syntax.css

```diff
@@ -77,3 +77,6 @@
 .highlight .vg { color: #008080 } /* Name.Variable.Global */
 .highlight .vi { color: #008080 } /* Name.Variable.Instance */
 .highlight .il { color: #009999 } /* Literal.Number.Integer.Long */
+
+/* Missing YAML styles */
+.highlight .l-Scalar-Plain { color: #008080 }
```

Comment:
Yeah, the highlighter we use doesn't have any built-in styling for YAML. I had to search around quite a bit for context and came up mostly dry.

Moreover, the actual tokenizer it uses only emits two token types, l-Scalar-Plain and p-Indicator. (You can see this in action by inspecting the HTML [here](https://ci.apache.org/projects/flink/flink-docs-release-1.2/monitoring/metrics.html#jmx-orgapacheflinkmetricsjmxjmxreporter))

This change applies the color I found most applicable from the existing highlighting rules and applies it to l-Scalar-Plain to make it at least slightly more interesting than black-on-white.